doc/architecture/decisions/0003-redux-architecture-pattern.md

```diff
@@ -0,0 +1,36 @@
+# 3. Redux architecture pattern
+
+Date: 2018-05-16
+
+## Status
+
+Accepted
+
+## Context
+
+We are doing a rebuild.
```

Comment:
Change to: We are doing a rewrite (see see ADR:0002)

Reply:
Am changing all of these right, though: I perceive rewrite and rebuild (in this context) to have the same meaning.